src/libos/src/fs/fs_ops/mount.rs

```diff
@@ -27,5 +30,26 @@ pub fn do_mount_rootfs(
         *root_inode = new_root_inode;
         *ENTRY_POINTS.write().unwrap() = user_config.entry_points.to_owned();
     });
+    // Write resolv.conf file into mounted file system
+    write_resolv_conf()?;
+
+    Ok(())
+}
+
+fn write_resolv_conf() -> Result<()> {
+    const RESOLV_CONF_PATH: &'static str = "/etc/resolv.conf";
+    let fs_view = FsView::new();
+    match fs_view.lookup_inode(RESOLV_CONF_PATH) {
+        Err(e) if e.errno() == ENOENT => {
+            let resolv_conf_file = fs_view.open_file(
+                RESOLV_CONF_PATH,
+                AccessMode::O_RDWR as u32 | CreationFlags::O_CREAT.bits(),
+                0o666,
+            )?;
+            resolv_conf_file.write(&*RESOLV_CONF_BYTES.read().unwrap().as_bytes());
+        }
+        Err(e) => return Err(e),
+        _ => (),
```

Comment:
I think we need to give more thought about this case of existing `/etc/resolv.conf` file.

In my opinion, it would be more robust to update the `/etc/resolv.conf` file on SEFS every time the LibOS starts. This way, if the network configuration of the host changes, the LibOS will use the latest configuration on startup.

With that said, I think the current design decision of not overriding existing `/etc/resolv.conf` file also makes sense in some use cases. But if whether overriding existing `/etc/resolv.conf` file is not a configurable policy (as least for now it is not), I would vote for the former approach, which is more robust in face of changing network configurations on host.

Reply:
Done, I had discussed that /etc/resolv.conf may be multi-times copied with Zongmin, we thought overriding the older /etc/resolv.conf is more reasonalble. Good advice!